app/breast-screening-pathway/2025/10/index.md

```diff
@@ -0,0 +1,247 @@
+| ``SI``	| Clinical processing, DNA episodes, mammography data entry, results entry, close routine episodes, print ad-hoc forms.
+| ``SCL``	| Clinics functionality, including clinic diaries, create/print clinics, rebook appointments.
+| ``SDY`` |	Daybook functionality like exporting a worklist or importing a daybook summary.
+| ``SF``	| Capture film reader overall results, individual client entry, PERFORMS worklist, or image assessment review.
+| ``ST``	| Receive and transmit data between NHAIS.
+| ``SG``	| Manages/prints GP (practice) details.
+| ``SL``	| Adhoc label/legacy printing, configure letter tasks.
+| ``SSM``	| Client search entry, film tracking/destruction, privileged client options, resend PACS messages associate clients to trials.
+| ``SM``	| Miscellaneous tasks like activity prompts and reprinting letters.
+| ``SP``	| Print adhoc reports, call/recall/ntd lists, or report schedules/tasks.
+| ``STA``	| Statistics and tables: AGEX, ASSEX, BASOX, CREGX, DLEX, SEXA, KC62 (single client prints/Dept. Health/cytology/histology/WBN QA/lymph node), film reader QA
+
+### User journey data discovery
+
+As users navigate through the system, there are many places they can manipulate, view or print data. To highlight the extent of these data points across the system, we identified several areas involved with data, as shown below:
+
+![Discovery Metrics](009_analysis_metrics.png)
+
+User journey data metrics
+
+<br/>
+
+|Category | Description |
+|-|-|
+| Menus | Starting function codes to access system functionality. These functions codes are configured separately for standard or administrator user types. |
+| Forms | Unique visual input forms to enter and view information. |
+| Screens | Simple navigation views or print parameters forms. |
+| Reports | Structured report templates that are either <li>``ad-hoc`` - custom user reports, <li>``standard`` - used by most services, <li>``regional`` - higher-level for a region, <li>``built-in`` - defined in the system by default. |
+| Letters | Structured report templates printed for clients. |
+| Data tables | Physical object storage locations. |
+| Data routines | Program code to create, update or transform NBSS data. |
+| Data classes | Program structures that typically represents tables. |
+
+---
+
+### Next Steps
+The Pathway Team will continue to
+* index all system knowledge, data and processes to provide a workable, queryable index that allows us to reason about the system without hindrance.
+* support a usable glossary of function code, form, screen, report, letter, class, routine, globals.
+* outline journey exemplars (SAD, SMSTA) supported by the index.
+
+
+[^1]: [What we learned - Improved data requirements](https://design-history.prevention-services.nhs.uk/breast-screening-pathway/2025/09/what-we-learned-about-breast-screening-data/)
+
+[^2]: [Assessment framework - How? What? Why?](https://nhsd-confluence.digital.nhs.uk/spaces/DTS/pages/1128218608/NBSS+Assessment)
```

Comment:
Unsure about linking to internal things like this - I wonder if at the least it should have some brackets to indicate it won't be publicly accessible.